Source/Tests/run_tests.py

```diff
@@ -38,10 +38,11 @@ def run_all_tests(labview_path: str):
 
 def _parse_command_line_args():
     parser = argparse.ArgumentParser(description="Run LabVIEW tests using LabVIEWCLI")
-    parser.add_argument("labview_path", type=str, help="Path to the LabVIEW executable", nargs='?', default=None)
+    parser.add_argument("--labview_path", type=str, help="Path to the LabVIEW executable", nargs='?', default=None)
+    parser.add_argument("--port_number", type=str, help="TCP/IP port number of LabVIEW executable", nargs='?', default="3363")
```

Comment:
Port numbers are numbers, not strings.

(FYI, some MeasurementLink code uses a string for the port in order to leave the door open to UNIX domain sockets, but that doesn't apply here.)